staging/src/k8s.io/client-go/rest/request.go

```diff
@@ -702,19 +729,6 @@ func (r *Request) request(fn func(*http.Request, *http.Response)) error {
 		metrics.RequestLatency.Observe(r.verb, r.finalURLTemplate(), time.Since(start))
 	}()
 
-	if r.err != nil {
```

Comment:
leave this in place.

Reply:
I have restored this to its original place on `request()` instead of `Do()` and `DoRaw()`. Coming up on the next commit.